backend/src/database.ts

```diff
@@ -16,3 +16,23 @@ export const dbDisconnect = async () => {
   await mongoose.disconnect();
   console.log("✅ Disconnected from MongoDB");
 };
+
+export function insertDoc() {
+  var db = mongoose.connection;
+
+  const Doc = mongoose.model('Doc', new Schema({
+    date: Number,
+    fog_net_id: String,
+    cluster_id: String,
+    water_collected: Number
+  }));
+
+  var doc1 = new Doc({
```

Comment:
In the future try to use `let` or `const` for variables
```suggestion
  let doc1 = new Doc({
```